diesel/src/expression/mod.rs

```diff
@@ -300,3 +300,9 @@ impl<QS, ST, DB> QueryId for BoxableExpression<QS, DB, SqlType = ST> {
 
     const HAS_STATIC_QUERY_ID: bool = false;
 }
+
+pub trait IntoSingleTypeExpressionList<ST> {
```

Comment:
I think this trait should also live in `diesel/src/pg/expression/array.rs` because it is Postgres specific.

Reply:
👍